### PR TITLE
Description and logo of be_piwikcharts added

### PR DESCRIPTION
**Pull request by mathContao**

### CODE REVIEW

**Review by fritzmg — CHANGES_REQUESTED** [review]
Please optimise the SVGs as described in the [README](https://github.com/contao/package-metadata/blob/main/docs/en/README.md).

**Comment by mathContao** [comment]
done. SVGs are optimized now

**Comment by fritzmg** [comment]
> done. SVGs are optimized now

Did you use https://jakearchibald.github.io/svgomg/ ? There are still line breaks in the SVG.

**Comment by mathContao** [comment]
> 
> 
> > done. SVGs are optimized now
> 
> Did you use https://jakearchibald.github.io/svgomg/ ? There are still line breaks in the SVG.

I used it.
Later I will check what went wrong

**Comment by fritzmg** [comment]
I think it would remove most of the whitespace usually, but may be I am remembering that wrong.

File: linter/allowlists/de.txt [hunk 1]
```diff
@@ -602,6 +602,8 @@ Vorschaubild
 Webanwendungen
 Webseite
 Webseitenwurzel
+Webseitenbesucher
```
Comment (aschempp) [inline linter/allowlists/de.txt:605]: Dieses Wort scheint nicht im Beschreibungstext vorzukommen?

**Review by aschempp — CHANGES_REQUESTED** [review]

File: linter/allowlists/de.txt (hunk 1 above)
Comment (mathContao) [reply linter/allowlists/de.txt:605]: It is the correct German word of "Websitebesucher". "Websitebesucher" was already in the file.

Comment (fritzmg) [reply linter/allowlists/de.txt:605]: Yes, but `Webseitenbesucher` is not used in your PR, so there is no need to add it. Please remove it again.

**Review by fritzmg — CHANGES_REQUESTED** [review]

**Review by fritzmg — APPROVED** [review]

**Review by aschempp — APPROVED** [review]

**Pull request merged**